discovery/gossiper.go

```diff
@@ -981,9 +981,9 @@ func (d *AuthenticatedGossiper) networkHandler() {
 
 	// To start, we'll first check to see if there are any stale channels
 	// that we need to re-transmit.
-	if err := d.retransmitStaleChannels(); err != nil {
-		log.Errorf("unable to rebroadcast stale channels: %v",
-			err)
+	err := d.retransmitStaleChannels()
+	if err != nil && err != channeldb.ErrGraphNoEdgesFound {
```

Comment:
Not sure if this works, as the error will be prepended with text inside `retransmitStaleChannels`. Instead we should check the error there, when querying the graph for channels, and return if no edges are found.

Reply:
Ah yeah, good catch!